resources/views/search/appetizer.blade.php

```diff
@@ -64,6 +66,56 @@ class="text-decoration-none"
                   @endif
                 @endauth
               </div>
+              <div class="col-1 d-lg-none ms-auto me-3">
+                <div class="dropdown">
+                  <button 
+                    class="btn
+                    dropdown-toggle btn-lg" 
+                    type="button" 
+                    id="dropdownMenuButton" 
+                    data-bs-toggle="dropdown" 
+                    aria-expanded="false"
+                    style="color: #887569; border: none;"
+                  >
+                  </button>
+                  <ul 
+                    class="dropdown-menu" 
+                    aria-labelledby="dropdownMenuButton"
+                  >
+                    <li  style="width: auto; margin-right: 1em; padding-right:0;">
+                      <a 
+                        class="dropdown-item" 
+                        href="{{ route('detailrecipe', ['post_id'=>$post->id, 'user_id'=>$post->user->id]) }}"
+                        style="color:e45900;"
+                      >
+                        &#xf086; 
+                        <span>{{ $post->comment->count() }}</span>
+                      </a>
+                    </li>
+                    <li  style="width: auto; margin-right: 1em; padding-right: 0;">
+                      @auth
+                        @if($post->isBookmarked())
+                          <a 
+                            class="dropdown-item" 
+                            href="{{ route('bookmark.toggle', ['post_id' => $post->id]) }}"
+                            style="color:e45900;"
```

Comment:
Prevent inline css. Make this a class instead. Since you are using the same code in other html tags